src/components/BurnIst.tsx

```diff
@@ -73,57 +73,48 @@ export default function BurnIst({ charterOfferId }: Props) {
       <h2 className="my-2 block text-lg leading-5 font-medium text-gray-700">
         Manual IST Burn
       </h2>
-      <motion.div
-        className="overflow-hidden px-1"
-        initial={{ height: 0, opacity: 0 }}
-        animate={{ height: 'auto', opacity: 1 }}
-        transition={{ type: 'tween' }}
-      >
-        <p className="my-1">{toBurnLabel}</p>
-        {brand && (
-          <AmountInput
-            disabled={isLoading}
-            value={currentInput}
-            brand={data?.shortfallBalance?.brand}
-            suffix="IST"
-            onChange={setCurrentInput}
-          ></AmountInput>
+      <p className="my-1">{toBurnLabel}</p>
+      <AmountInput
+        disabled={isLoading}
+        value={currentInput}
+        brand={data?.shortfallBalance?.brand}
+        suffix="IST"
+        onChange={setCurrentInput}
+      ></AmountInput>
+      <AnimatePresence>
+        {inputError && (
+          <motion.div
+            initial={{ height: 0 }}
+            animate={{ height: 'auto' }}
+            exit={{ height: 0 }}
+            transition={{ type: 'tween', duration: 0.25 }}
+            className="text-error overflow-hidden"
+          >
+            {inputError}
+          </motion.div>
         )}
-        <AnimatePresence>
-          {inputError && (
-            <motion.div
-              initial={{ height: 0 }}
-              animate={{ height: 'auto' }}
-              exit={{ height: 0 }}
-              transition={{ type: 'tween', duration: 0.25 }}
-              className="text-error overflow-hidden"
-            >
-              {inputError}
-            </motion.div>
+      </AnimatePresence>
+      <label className="block mt-2">
+        <span className="text-gray-700">Minutes until close of vote</span>
+        <input
+          type="number"
+          className="rounded mt-1 block w-full border-gray-300 focus:border-purple-300 focus:ring-purple-300"
+          value={minutesUntilClose}
+          onChange={e => setMinutesUntilClose(e.target.valueAsNumber)}
+        />
+      </label>
+      <div className="w-full flex flex-row justify-end mt-2">
+        <button
+          className={clsx(
+            'btn-primary px-3 py-2 rounded mt-2',
+            canMakeProposal ? 'cursor-pointer' : 'cursor-not-allowed',
```

Comment:
I know this is just a refactor but it prompted me to search and I see 8 instances of this kind of button. Seems worth its own component. Not a blocker.